### ISSUE TITLE
@neovim.function doesn't work?

### ISSUE THREAD

**Issue by raghur** [neovim/python-client#280]
I'm trying to write a plugin - here's the code https://github.com/raghur/vim-ghost/blob/master/rplugin/python3/ghost.py with a function

```
@neovim.function("ghostSend", sync=True)
    def ghostSend(self, args):
logging.info(args)
```

However, while the commands are available, the function isn't available while doing `:call gh<tab>`. 
I have run UpdateRemotePlugins and the rplugin.vim has the function registration as well.

Is there something else needed to make the function available?

**Comment by justinmk** [comment]
Does `:call ghostSend()` work?

**Comment by bfredl** [comment]
It must start uppercase: GhostSend()

**Comment by raghur** [comment]
>It must start uppercase: GhostSend()
Func autocomplete did not show it - but when I called it, there were no errors. Still not seeing a log though (I've set `export NVIM_PYTHON_LOG_FILE=~/pylog` and I see log files for nvim completion manager etc being created but nothing for my plugin.

@justinmk : Nope - indentation looks good to me :confused:  and I dont have any lint errors on that line.. Looks like func name has to be uppercase. With lowercase first letter, I get unknown function.
